### ISSUE TITLE
No image in manuals

### ISSUE THREAD

**Issue by IRA1777** [joe7575/techage#153]
Images are not displayed in the github manuals.

**Comment by joe7575** [comment]
Yes I know. The images are primarily for in-game documentation. So this is not a bug, at most a feature request. But I wouldn't know how to implement it straight away.
